### ISSUE TITLE
yubikey-manager (ykman) fails with an error message about too many arguments passed to list_devices()

### ISSUE THREAD

**Issue by jhe2** [void-linux/void-packages#29810]
### System

* xuname:  Void 5.11.4_1 x86_64 GenuineIntel uptodate hold rrrmFFF
* package:  yubikey-manager-3.1.1_5

### Expected behavior

``ykman fido info`` should display status information, when a FIDO2 compatible stick is attached.

### Actual behavior

An error message is printed instead:

```
/usr/lib/python3.9/site-packages/ykman/opgp.py:40: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_to_bytes, int_from_bytes
/usr/lib/python3.9/site-packages/ykman/piv.py:39: CryptographyDeprecationWarning: int_from_bytes is deprecated, use int.from_bytes instead
  from cryptography.utils import int_to_bytes, int_from_bytes
Traceback (most recent call last):
  File "/usr/bin/ykman", line 33, in <module>
    sys.exit(load_entry_point('yubikey-manager==3.1.1', 'console_scripts', 'ykman')())
  File "/usr/lib/python3.9/site-packages/ykman/cli/__main__.py", line 273, in main
    cli(obj={})
  File "/usr/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python3.9/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/usr/lib/python3.9/site-packages/ykman/cli/fido.py", line 87, in info
    controller = ctx.obj['controller']
  File "/usr/lib/python3.9/site-packages/ykman/cli/util.py", line 127, in __getitem__
    self.resolve()
  File "/usr/lib/python3.9/site-packages/ykman/cli/util.py", line 124, in resolve
    self._objects[k] = f()
  File "/usr/lib/python3.9/site-packages/ykman/cli/__main__.py", line 194, in resolve_device
    dev = _run_cmd_for_single(ctx, subcmd.name, transports, reader)
  File "/usr/lib/python3.9/site-packages/ykman/cli/__main__.py", line 132, in _run_cmd_for_single
    return descriptor.open_device(transports)
  File "/usr/lib/python3.9/site-packages/ykman/descriptor.py", line 96, in open_device
    for drv in _list_drivers(transports):
  File "/usr/lib/python3.9/site-packages/ykman/descriptor.py", line 164, in _list_drivers
    for dev in open_fido():
  File "/usr/lib/python3.9/site-packages/ykman/driver_fido.py", line 97, in open_devices
    for dev in CtapHidDevice.list_devices(descriptor_filter):
TypeError: list_devices() takes 1 positional argument but 2 were given
```

### Steps to reproduce the behavior

* Install the ``yubikey-manager`` package
* Run ``ykman fido info`` with a Yubikey attached
* Observe error message

The tool appears to be working fine if there is no compatible device attached, without any indication of a problem.

The latest ``yubikey-manager`` (which is version 4.0.0 at the moment) installed via ``pip`` works without issue on the same system.

**Comment by FollieHiyuki** [comment]
The problem is probably because of `python3-fido2` (which is fixed in 3.1.2).
Updating the package to 4.0.0 would be enough.

**Comment by sgn** [comment]
Fixed with e904ed8170
